index.md

```diff
@@ -3,50 +3,60 @@ layout: lesson
 root: .
 ---
 
-Wolfman and Dracula have been hired by Universal Missions (a space
```

Comment:
This will overwrite the homepage.

Reply:
Yes, I agree. The rest of the files would be ok, right @rgaiacs ?

Reply:
I'm not familiar with transifex and the internals of the build, but what if we move all files to a language folder and keep the same structure? So `_episodes_es` -> `lang/es/_episodes/` and instead of overwriting the `index.md`, we move the spanish version to `lang/es/index.md`.

Would jekyll build this pages? If so, we could modify the workshop template to add a pane for alternative languages that simply links to `ES` to `/lang/es/index.html`. Jekyll could crawl thru the `lang` folder is the base of the repo and set up the links for available languages. 

Would that be a fix, @rgaiacs ?